src/main/resources/allowed-labels.properties

```diff
@@ -38,6 +40,7 @@ gitlab
 groovy-related
 google-cloud
 ios
+jira
```

Comment:
Plugins that seemed like good candidates for this label include:

* [Atlassian Jira software cloud](https://plugins.jenkins.io/atlassian-jira-software-cloud/)
* [Jira](https://plugins.jenkins.io/jira/)
* [Test management for Jira](https://plugins.jenkins.io/tm4j-automation/)
* [X-ray connector for Jira](https://plugins.jenkins.io/xray-connector/)
* [Jira pipeline steps](https://plugins.jenkins.io/jira-steps/)
* [Jira issue updater](https://plugins.jenkins.io/jenkins-jira-issue-updater/)
* [Zephyr for Jira test management](https://plugins.jenkins.io/zephyr-for-jira-test-management/)
* [QMetry for Jira test management](https://plugins.jenkins.io/qmetry-for-jira-test-management/)
* [Bitbucket Jira validator SCM filter](https://plugins.jenkins.io/bitbucket-scm-filter-jira-validator/)
* [GitHub Jira Validator SCM filter](https://plugins.jenkins.io/github-scm-filter-jira-validator/)
* [Jira integration for Blue Ocean](https://plugins.jenkins.io/blueocean-jira/)
* [Jira test result reporter](https://plugins.jenkins.io/JiraTestResultReporter/)
* [iTMS for Jira](https://plugins.jenkins.io/itms-for-jira/)
* [Release Helper](https://plugins.jenkins.io/release-helper/)
* [Generic Webhook trigger](https://plugins.jenkins.io/generic-webhook-trigger/)

Jira is a great candidate for a label.  Are there plugins other than those which should receive the `jira` label?